clippy_lints/src/redundant_deref.rs

```diff
@@ -0,0 +1,63 @@
+// use crate::utils::{get_parent_expr, snippet_with_applicability, span_lint_and_sugg};
```

Comment:
We don't usually commit commented-out code. Is there a reason for this or is it just a leftover?

Reply:
That is an accident, yes.